packages/contracts-bedrock/contracts/universal/FeeVault.sol

```diff
@@ -65,12 +88,16 @@ abstract contract FeeVault {
         uint256 value = address(this).balance;
         totalProcessed += value;
 
-        emit Withdrawal(value, RECIPIENT, msg.sender);
+        emit Withdrawal(value, RECIPIENT, msg.sender, WITHDRAWAL_NETWORK);
 
-        L2StandardBridge(payable(Predeploys.L2_STANDARD_BRIDGE)).bridgeETHTo{ value: value }(
-            RECIPIENT,
-            WITHDRAWAL_MIN_GAS,
-            bytes("")
-        );
+        if (WITHDRAWAL_NETWORK == WithdrawalNetwork.L2) {
+            SafeCall.call(RECIPIENT, gasleft(), value, bytes(""));
```

Comment:
Should use `SafeCall.send`
https://github.com/ethereum-optimism/optimism/blob/75c00a5091e43cc50d9b33381a166c68c6a465cd/packages/contracts-bedrock/contracts/libraries/SafeCall.sol#L17